classes/core/PKPRequest.inc.php

```diff
@@ -591,6 +591,18 @@ function &getUser() {
 	function getUserVar($key) {
 		$_this = PKPRequest::_checkThis();
 
+		if (is_a($this, 'Request')) {
```

Comment:
The `$this` / `$_this` handling here is historical: all calls to Request functions used to be static, and in some cases calls deeper in the software stack don't have access to a `$request` instance. So we're gradually moving to instance calls but some static calls still remain. Is there a reason we can't stick with `$_this` while that's being cleaned up? It would be more consistent.